src/core/thread/thread_uris.hpp

```diff
@@ -40,224 +40,240 @@ namespace Thread {
  *
  */
-#define OPENTHREAD_URI_ENERGY_REPORT          "c/er"
+#define OPENTHREAD_URI_ENERGY_REPORT            "c/er"
 
 /**
  * @def OPENTHREAD_URI_PENDING_GET
  *
  * The URI Path for MGMT_PENDING_GET
  *
  */
-#define OPENTHREAD_URI_PENDING_GET            "c/pg"
+#define OPENTHREAD_URI_PENDING_GET              "c/pg"
 
 /**
  * @def OPENTHREAD_URI_PENDING_SET
  *
  * The URI Path for MGMT_PENDING_SET
  *
  */
-#define OPENTHREAD_URI_PENDING_SET            "c/ps"
+#define OPENTHREAD_URI_PENDING_SET              "c/ps"
 
 /**
  * @def OPENTHREAD_URI_SERVER_DATA
  *
  * The URI Path for Server Data Registration.
  *
  */
-#define OPENTHREAD_URI_SERVER_DATA            "a/sd"
+#define OPENTHREAD_URI_SERVER_DATA              "a/sd"
 
 /**
  * @def OPENTHREAD_URI_ANNOUNCE_BEGIN
  *
  * The URI Path for Announce Begin.
  *
  */
-#define OPENTHREAD_URI_ANNOUNCE_BEGIN         "c/ab"
+#define OPENTHREAD_URI_ANNOUNCE_BEGIN           "c/ab"
 
 /**
  * @def OPENTHREAD_URI_RELAY_RX
  *
  * The URI Path for Relay RX.
  *
  */
-#define OPENTHREAD_URI_RELAY_RX               "c/rx"
+#define OPENTHREAD_URI_RELAY_RX                 "c/rx"
 
 /**
  * @def OPENTHREAD_URI_RELAY_TX
  *
  * The URI Path for Relay TX.
  *
  */
-#define OPENTHREAD_URI_RELAY_TX               "c/tx"
+#define OPENTHREAD_URI_RELAY_TX                 "c/tx"
 
 /**
  * @def OPENTHREAD_URI_JOINER_FINALIZE
  *
  * The URI Path for Joiner Finalize
  *
  */
-#define OPENTHREAD_URI_JOINER_FINALIZE        "c/jf"
+#define OPENTHREAD_URI_JOINER_FINALIZE          "c/jf"
 
 /**
  * @def OPENTHREAD_URI_JOINER_ENTRUST
  *
  * The URI Path for Joiner Entrust
  *
  */
-#define OPENTHREAD_URI_JOINER_ENTRUST         "c/je"
+#define OPENTHREAD_URI_JOINER_ENTRUST           "c/je"
 
 /**
  * @def OPENTHREAD_URI_LEADER_PETITION
  *
  * The URI Path for Leader Petition
  *
  */
-#define OPENTHREAD_URI_LEADER_PETITION        "c/lp"
+#define OPENTHREAD_URI_LEADER_PETITION          "c/lp"
 
 /**
  * @def OPENTHREAD_URI_LEADER_KEEP_ALIVE
  *
  * The URI Path for Leader Keep Alive
  *
  */
-#define OPENTHREAD_URI_LEADER_KEEP_ALIVE      "c/la"
+#define OPENTHREAD_URI_LEADER_KEEP_ALIVE        "c/la"
 
 /**
  * @def OPENTHREAD_URI_PANID_CONFLICT
  *
  * The URI Path for PAN ID Conflict
  *
  */
-#define OPENTHREAD_URI_PANID_CONFLICT         "c/pc"
+#define OPENTHREAD_URI_PANID_CONFLICT           "c/pc"
 
 /**
  * @def OPENTHREAD_URI_PANID_QUERY
  *
  * The URI Path for PAN ID Query
  *
  */
-#define OPENTHREAD_URI_PANID_QUERY            "c/pq"
+#define OPENTHREAD_URI_PANID_QUERY              "c/pq"
 
 /**
  * @def OPENTHREAD_URI_COMMISSIONER_GET
  *
  * The URI Path for MGMT_COMMISSIONER_GET
  *
  */
-#define OPENTHREAD_URI_COMMISSIONER_GET       "c/cg"
+#define OPENTHREAD_URI_COMMISSIONER_GET         "c/cg"
 
 /**
  * @def OPENTHREAD_URI_COMMISSIONER_SET
  *
  * The URI Path for MGMT_COMMISSIONER_SET
  *
  */
-#define OPENTHREAD_URI_COMMISSIONER_SET       "c/cs"
+#define OPENTHREAD_URI_COMMISSIONER_SET         "c/cs"
 
```

Comment:
I will adjust the position of the URI according to the alphabetical order. Thanks.